docs/book/known_issues.md

```diff
@@ -46,3 +46,7 @@ Issue 5<a id="issue_5"></a>: The CSI delete volume is getting called before deta
     4. Edit this VolumeAttachment and remove the finalizer. It will get deleted.
     5. Use `govc` to manually delete the FCD.
     6. Edit Pending PV and remove the finalizer. It will get deleted.
+
+Issue 6<a id="issue_6"></a>: vSphere with Kubernetes Cluster Devops can modify the volume health status of a PVC manually since the volume health annotation is not a read-only field. Devops should avoid modifying the volume health annotation manually. If DevOps modifies the volume health to a random or incorrect health status, then any software dependent on this volume health will be affected.
+
+- Impact: Any random volume health status value that set by the Kubernetes Cluster Devops will get reflected in volume health status of PVC in Tanzu Kubernetes Grid Cluster as well.
```

Comment:
Change it to- Any random volume health status set by the vSphere with Kubernetes Cluster Devops...